provider/k8s.go

```diff
@@ -1061,18 +1070,14 @@ func (k8s *K8sOfflineStore) pandasRunnerArgs(outputURI string, updatedQuery stri
 }
 
 func (k8s K8sOfflineStore) getDFArgs(outputURI string, code string, mapping []SourceMapping, sources []string) map[string]string {
-	// mappingSources := make([]string, len(mapping))
-	// for i, item := range mapping {
-	// 	mappingSources[i] = item.Source
-	// }
 	sourceList := strings.Join(sources, ",")
 	envVars := map[string]string{
 		"OUTPUT_URI":          outputURI,
 		"SOURCES":             sourceList,
 		"TRANSFORMATION_TYPE": "df",
 		"TRANSFORMATION":      code,
 	}
-	_, ok := k8s.executor.(KubernetesExecutor)
+	_, ok := k8s.executor.(*KubernetesExecutor)
```

Comment:
You could merge this with the line below

Reply:
Changed the other one as well